WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationCommentDetailViewController.swift

```diff
@@ -284,18 +310,18 @@ private extension NotificationCommentDetailViewController {
         return nil
     }
 
-    func fetchComment(_ commentID: NSNumber?, completion: @escaping (Comment?) -> Void) {
+    func fetchComment(_ commentID: NSNumber?, completion: @escaping (Comment?) -> Void, failure: @escaping (Error?) -> Void) {
         guard let commentID = commentID else {
             DDLogError("Notification Comment: unable to fetch comment due to missing commentID.")
-            completion(nil)
+            failure(nil)
             return
         }
 
         if let blog = blog {
             commentService.loadComment(withID: commentID, for: blog, success: { comment in
                 completion(comment)
             }, failure: { error in
-                completion(nil)
+                failure(error)
```

Comment:
Does this display BE error directly? If so, that won't be localised and may not make sense for the user.